agenta-cli/agenta/cli/main.py

```diff
@@ -101,9 +101,15 @@ def init(app_name: str):
                 "Please provide the IP or URL of your remote host"
             ).ask()
         elif where_question == "On agenta cloud":
-            backend_host = "https://cloud.agenta.ai"
+            which_agenta = questionary.select(
```

Comment:
I don't agree with this solution. We should not add friction to our end users to simplify our development. A better solution would be to just load this from the global config file. So that when you develop you modify the value of the backend host in the global config file from cloud.agenta.ai to whatever host you would like

Reply:
Ah, I see. Thank you for letting me know.